roles/analysis/handlers/main.yml

```diff
@@ -6,4 +6,10 @@
     activationkey: "{{ satellite_activation_key_pre_leapp }}"
     org_id: "{{ satellite_organization }}"
     force_register: true
+
+- name: Remove lynx package
```

Comment:
@jeffmcutter, why are we removing this package during analysis? Ideally, the analysis role shouldn't be making changes beyond installing the packages required to generate the report.

Reply:
Got it, thanks!